cernopendata/modules/fixtures/data/records/cms-corrections-run2.json

```diff
@@ -0,0 +1,61 @@
+[
+  {
+    "abstract": {
+      "description": "<p>CMS supports corrections for various physics objects, either to improve modeling of data by simulation, or to correct for imperfect detector response. For Run 2 data, the official corrections are stored in a common JSON format that can be read with the correctionlib library.</p>"
+    },
+    "accelerator": "CERN-LHC",
+    "authors": [
+      {
+        "name": "CMS Open data group"
+      }
+    ],
+    "collections": [
+      "CMS-Tools"
+    ],
+    "date_created": [
+      "2016"
+    ],
+    "date_published": "2024",
+    "distribution": {
+      "formats": [
+        "zip"
+      ],
+      "number_files": 1,
+      "size": 10494603
+    },
+    "experiment": [
+      "CMS"
+    ],
+    "files": [
+      {
+        "checksum": "adler32:7ccaf136",
+        "size": 10494603,
+        "uri": "root://eospublic.cern.ch//eos/opendata/cms/corrections/jsonpog-integration-2016post.tar"
+      }
+    ],
+    "license": {
+      "attribution": "CC0"
+    },
+    "publisher": "CERN Open Data Portal",
+    "recid": "12508",
```

Comment:
> I chose an unused recid that is in our "tools" group. Might be nice to adjust to a different number range? We will probably end up with O(5) of these records for "Corrections".

I kept the record ID 12508 for now, but if you wish we could find a place "closer" to the Run2 NanoAOD records. Will these new 5 records concern Run2?

Reply:
@tiborsimko even beyond, so maybe just best to leave it -- probably this record for Run 2, one for Run 3, maybe one for heavy ions at some point (?), maybe one for the correctionlib software...